apps/glusterfs/app_volume.go

```diff
@@ -95,6 +95,18 @@ func (a *App) VolumeCreate(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	for _, volumeOption := range msg.Options {
+		switch volumeOption {
+		case "encryption":
```

Comment:
@humblec There is not much of a discussion about a whitelist in #711 actually. In fact the only argument to be read there is that it would be *bad* to maintain an additional wrapping layer in heketi, because it would require us to keep pace with gluster changes and options in heketi. (see @jarrpa 's comment https://github.com/heketi/heketi/issues/711#issuecomment-285668834).

I originally thought a higher level set of options that wrap a couple of gluster options would be good to have, and you said you wanted a whitelist of gluster options instead. Then José kind of convinced me that we should not add a complicated wrapping layer but mainly hand options through verbatim. But the current patch does actually not add a whitelist, but it adds a higher level option "encryption" which will translate into setting ```server.ssl``` and ```client.ssl``` to on.

I am not convinced yet that this is the way to go. I think there are several aspects to adding options:

* convenience and simplicity of use
* safety
* maintenance
* flexibility

Going with higher level wrapping and heketi-level checking will:

* make it convenient, simple to use
* will be safe to use in a sense that heketi prevents inconsistent setting
* will make maintenance much harder (gluster changes imply heketi changes)
* will be totally unflexible (need to do code changes for every new option we want to pass)

On the other hand side, just passing options through to gluster verbatim will:

* be less convenient to use (need to write out all options)
* require the admin to know more about gluster
* will be only slightly less safe to use (checking done by gluster)
* will require zero maintenance (gluster option changes do NOT require heketi changes)
* will be very flexible (using new options does NOT require heketi changes)

Based on these, I would now vote for doing the verbatim passing-through.
Minimal code change with maximum flexibility. KISS.

Of course we might think about adding some specific high-level / heketi-level
options like "encryption" in parallel. Just need to decide about the names and
about which takes precedence over what...